Formula/meta-package-manager.rb

```diff
@@ -0,0 +1,417 @@
+class MetaPackageManager < Formula
+  include Language::Python::Virtualenv
+
+  desc "🎁 wraps all package managers with a unifying CLI"
+  homepage "https://github.com/kdeldycke/meta-package-manager"
```

Comment:
Following what's shown on the [GitHub repo page](https://github.com/kdeldycke/meta-package-manager).

```suggestion
  homepage "https://kdeldycke.github.io/meta-package-manager/"
```

Reply:
Good to me! Let's merge this change!